lib/stat_tracker.rb

```diff
@@ -103,14 +103,6 @@ def goals_per_game_per_team
     end
   end
 
-  def total_shots_per_team
-    @game_teams.reduce(Hash.new(0)) do |result, game_team|
-      result[game_team.team_id] += game_team.shots
-      result
-    end
-  end
-
-
   def games_won_per_team_per_season(season_id)
     @game_teams.reduce(Hash.new(0)) do |result, game_team|
```

Comment:
This method should be deleted now - same as the one on line 176. Please ctrl+F this games_won_per_team_per_season to ensure it's not being used anywhere else - if so update the name. Same goes for the test.